### ISSUE TITLE
Dropout

### ISSUE THREAD

**Issue by munsanje** [rykov8/ssd_keras#70]
Has anyone successfully applied dropout to the architecture? I get bizzare results, such as very low training and validation loss but poor performance on the test set. I'm applying dropout before each pooling layer as well as on the last convolutional layers of each map used to extract features by the fully connected layers.

**Comment by shuuchen** [comment]
@munsanje 
Dropout is removed according to the original paper.
